src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval.java

```diff
@@ -778,10 +833,28 @@ public JSON denyApprovedClasspathEntry(String hash) throws IOException {
     @Restricted(NoExternalUse.class) // for use from AJAX
     @JavaScriptMethod
     public synchronized JSON clearApprovedClasspathEntries() throws IOException {
-        Jenkins.getInstance().checkPermission(Jenkins.RUN_SCRIPTS);
+        final Jenkins jenkins = getJenkins();
+        jenkins.checkPermission(Jenkins.RUN_SCRIPTS);
         approvedClasspathEntries.clear();
         save();
         return getClasspathRenderInfo();
     }
 
+    @Nonnull
+    private static Jenkins getJenkins() throws IOException {
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins == null) {
+            throw new IOException("Jenkins instance is not ready");
+        }
+        return jenkins;
+    }
+    
+    @Nonnull
+    private static Jenkins getActiveJenkinsInstance() throws IllegalStateException {
```

Comment:
Should have a `TODO` comment to remove once the baseline supports `Jenkins.getActiveInstance`.